utils.py

```diff
@@ -332,10 +334,19 @@ def are_datetimes_close(later_datetime, earlier_datetime):
 
 
 def generate_random_string(length):
+    """Generates a random string of the specified length.
+
+    Args:
+      length(Int): Length of the string to be generated.
+
```

Comment:
Its fine but the correct format is-- **length: int. Length of the string to be generated.** See my previous comment. I wrote there very clearly.